site/src/components/series/point.md

```diff
@@ -14,27 +14,61 @@ example2-code: |
       .datum(data)
       .call(point);
 
-example-code: |
+example-code-2: |
   var color = d3.scale.category10();
 
   var point = fc.series.point()
       .xScale(xScale)
       .yScale(yScale)
-      .radius(function(d, i) { return i; })
+      .size(function(d, i) { return i * i * 3.14; }) //Size is specified in pixels area
       .decorate(function(sel) {
-        sel.select('circle')
+        sel.select('path')
           .attr('style', function(d, i) { return 'fill: ' + color(i); });
       });
 
   container.append('g')
       .datum(data)
       .call(point);
+      
+example-code-3: |
+  var point = fc.series.point()
+      .xScale(xScale)
+      .yScale(yScale)
+      .type(function(d, i) {
+          var type;
+          switch(i%6) {
+              case 0:
+                  type = 'circle';
+                  break;
+              case 1:
+                  type = 'cross';
+                  break;
+              case 2:
+                  type = 'diamond';
+                  break;
+              case 3:
+                  type = 'square';
+                  break;
+              case 4:
+                  type = 'triangle-down';
+                  break;
+              case 5:
+                  type = 'triangle-up';
+                  break;          
+          }
+          return type;
+      });
+      
+  container.append('g')
+        .datum(data)
+        .call(point);
+      
 ---
 
-The point series renders the given data as a series of circles:
+The point series is based on the `d3.svg.symbol` element. By default it renders the given data as a series of circles:
```

Comment:
Something like:
The point series renders the given data as series of symbols. The symbols are based on the `d3.svg.symbol` element. By default, it renders circles with a radius of 5 pixels. This symbol can be changed using the `type` property, and resized using the `size` property.